File: cache/cache-annotation-processor/src/test/java/ru/tinkoff/kora/cache/annotation/processor/testcache/DummyCache12.java
package ru.tinkoff.kora.cache.annotation.processor.testcache;

import ru.tinkoff.kora.cache.annotation.Cache;
import ru.tinkoff.kora.cache.caffeine.CaffeineCache;
import ru.tinkoff.kora.cache.redis.RedisCache;

@Cache("dummy2")
public interface DummyCache12 extends RedisCache<String, String> {

}

File: cache/cache-symbol-processor/src/test/kotlin/ru/tinkoff/kora/cache/symbol/processor/testcache/DummyCache12.kt
package ru.tinkoff.kora.cache.symbol.processor.testcache

import ru.tinkoff.kora.cache.annotation.Cache
import ru.tinkoff.kora.cache.caffeine.CaffeineCache
import ru.tinkoff.kora.cache.redis.RedisCache

@Cache("dummy12")
interface DummyCache12 : RedisCache<String, String>

